web/README.md

```diff
@@ -50,7 +50,7 @@ Currently runs with:
 
 - Angular v5.2.5
 - Angular-CLI v1.6.4
-- Electron v1.8.7
+- Electron v1.8.2
```

Comment:
Electron is now at 1.8.7, Angular is now at 6.0.9 and CLI is at 6.0.8
🎆 😃

Reply:
Thanks I will change that